website/docs/developer/research/research-bamboo.md

```diff
@@ -153,7 +153,7 @@ Code for the prototype resides currently in this [branch](https://github.com/nas
 ## Future directions
 
 - Convert bamboo plans to YAML definition language and store under `cumulus-orca/bamboo-specs/bamboo.yaml`.
-- Discuss with team about enabling Webhooks in ORCA github repo. 
+- A NASD ticket will need to be submitted in order for them to add the webhook to the `cumulus-orca` repo.
```

Comment:
Adjust this to email and point to instructions sent in chat. Will wee need to do this every time we make changes or is this a 1 and done? By changes I mean new or different plans.

Reply:
should be just one time. Once webhook is added, I am guessing we don't have to make anymore changes. 

Updated.